components/SearchForm/index.js

```diff
@@ -25,39 +83,87 @@ const EmployeeProfile = () => {
     setSearchValue("");
   }
 
+  function handleSort(event) {
+    const sort = event.target.value;
+    setSortOrder(sort);
+  }
+
+  let sortedEmployees = [...(searchValue ? searchResults : employees)];
+
+  // Sortiert die Mitarbeiter basierend auf der ausgewählten Sortierreihenfolge
+  if (sortOrder === "name") {
+    sortedEmployees.sort((a, b) => a.name.localeCompare(b.name));
+  } else if (sortOrder === "position") {
+    sortedEmployees.sort((a, b) => a.position.localeCompare(b.position));
+  } else if (sortOrder === "age") {
+    sortedEmployees.sort((a, b) => a.age - b.age);
+  } else if (sortOrder === "startDate") {
+    sortedEmployees.sort(
+      (a, b) => new Date(a.startDate) - new Date(b.startDate)
+    );
+  }
+
   return (
     <>
-      <h2>Mitarbeiterliste</h2>
-      <form onSubmit={handleSearch}>
-        <label htmlFor="search_name">Suche:</label>
-        <input type="text" name="search_name" id="search_name" />
-        <button type="submit">Suchen</button>
-      </form>
-
-      <button onClick={handleShowAll}>Alle anzeigen</button>
-      <ul>
-        {(searchValue ? searchResults : employees).map((employee) => (
-          <li key={employee.id}>
-            <div>
-              <img
+      <h3>Startup Company</h3>
+
+      {/* Suchleiste */}
+      <SearchBarContainer onSubmit={handleSearch}>
+        <SearchInput type="text" name="search_name" id="search_name" />
+        <SearchButton type="submit">Suchen</SearchButton>
+        <ShowAllButton type="button" onClick={handleShowAll}>
+          Alle anzeigen
+        </ShowAllButton>
+      </SearchBarContainer>
+
+      {/* Sortier-Dropdown */}
+      <SortDropdownContainer>
+        <label htmlFor="sort_order">Sortieren nach:</label>
+        <select id="sort_order" name="sort_order" onChange={handleSort}>
+          <option value="name">Name</option>
+          <option value="age">Alter</option>
+          <option value="startDate">Startdatum</option>
+          <option value="position">Position</option>
+          <option value="department">Abteilung</option>
+          <option value="salary">Gehalt</option>
+        </select>
```

Comment:
you could style this select element (with styled components;))  to make its more pretty :)